dashboard/app/views/pd/workshop_mailer/_workshop_logistics.html.haml

```diff
@@ -31,3 +31,8 @@
           = facilitator.name
           (#{mail_to(facilitator.email)})
           %br
+  - if !@workshop.notes.blank? && @workshop.virtual?
+    %td.heading
+      Notes:
+    %td
+      = @workshop.notes
```

Comment:
Same as above, with this affecting CSA Capstone but also not specific to it and we want this change to be effective for all virtual workshops.